src/buffer.rs

```diff
@@ -948,6 +949,9 @@ where
 {
     /// Writes the buffer to a writer in the specified format.
     ///
+    /// Assumes the writer is buffered. In most cases,
+    /// you should wrap your writer in a `BufWriter` for best performance.
+    ///
```

Comment:
Documentation: Clarified whether the writer being buffering or not buffered while I was there